utils/utils.go

```diff
@@ -1374,34 +1375,29 @@ func FormatPoolPerformance(val float64) template.HTML {
 }
 
 func FormatTokenSymbolTitle(symbol string) string {
-	urls := xurls.Relaxed.FindAllString(symbol, -1)
-
-	if len(urls) > 0 {
-		return fmt.Sprintf("The token symbol has been hidden as it contains a URL (%s) which might be a scam", symbol)
-	} else if symbol == "ETH" {
-		return fmt.Sprintf("The token symbol has been hidden as it contains a Token name (%s) which might be a scam", symbol)
+	if isMaliciousToken(symbol) {
+		return fmt.Sprintf("The token symbol (%s) has been hidden because it contains a URL or a confusable character", symbol)
 	}
 	return ""
 }
 
 func FormatTokenSymbol(symbol string) string {
-	urls := xurls.Relaxed.FindAllString(symbol, -1)
-
-	if len(urls) > 0 ||
-		symbol == "ETH" {
+	if isMaliciousToken(symbol) {
 		return "[hidden-symbol] ⚠️"
 	}
 	return symbol
 }
 
+func isMaliciousToken(symbol string) bool {
+	urls := xurls.Relaxed.FindAllString(symbol, -1)
+	isConfusable := confusables.IsDangerous(symbol, []string{"latin"})
+	return len(urls) > 0 || isConfusable || symbol == "ETH"
+}
+
 func FormatTokenSymbolHTML(tmpl template.HTML) template.HTML {
-	tmplString := (string(tmpl))
+	tmplString := FormatTokenSymbol(string(tmpl))
 	symbolTitle := FormatTokenSymbolTitle(tmplString)
```

Comment:
**issue:** This doesn't work because the first line already changes the symbol from invalid "ETH" to "[hidden-symbol] ⚠️" and you input that into `FormatTokenSymbolTitle` which it doesn't have a problem with so it returns "".

Reply:
removed the function, see me answer to your last comment